subtests/docker_cli/create/create_tmpfs.py

```diff
@@ -0,0 +1,28 @@
+"""
+Create container with --tmpfs option and inspect it
+"""
+
+from create import create_base
+from dockertest.output import DockerVersion
+
+
+class create_tmpfs(create_base):
+
+    def initialize(self):
+        super(create_tmpfs, self).initialize()
+        # Minimum docker version 1.10 is required for --tmpfs
+        DockerVersion().require_server("1.10")
+
+    def run_once(self):
+        super(create_tmpfs, self).run_once()
+        cid = self.get_cid()
+        cont_long_id = self.sub_stuff['cont'].json_by_long_id(cid)
```

Comment:
Much better! But I don't understand this variable name. `json_by_long_id` returns a dict, parsed from the json returned by `docker inspect`. I don't think the `long_id` part is meaningful to the contents of the data structure or the variable. I think a better name would reflect the fact that this is docker inspect contents. But I may have missed something; please let me know if you have a good reason for this var name.